devices/2.json

```diff
@@ -1,6 +1,20 @@
 {"mdl":"Smart Plug+",
     "parameters":[
-        {"id": "switch",        "component":"switch", "stat_t":"~@/state","cmd_t":"~@/command", "config": {"icon":"mdi:power-socket-uk","opt":false}},
-        {"id": "REAL_POWER",    "component":"sensor", "stat_t":"~@/state","config": {"dev_cla":"power","unit_of_meas":"W"}},
-        {"id": "REACTIVE_POWER","component":"sensor", "stat_t":"~@/state","config": {"dev_cla":"reactive_power","unit_of_meas":"var"}}    ]
+        {"id": "switch", "component":"switch", "stat_t":"~@/state", "cmd_t":"~@/command", "config": {
+            "name":"Switch",
+            "dev_cla":"outlet",
+            "icon":"mdi:power-socket-uk",
+            "opt":false,
+            "payload_on":"ON",
```

Comment:
payload_on, payload_off, state_on and state_off are all default values and shouldn't be specified here

Reply:
Also see comments for 1.json and apply here

Reply:
remove id "Switch" it is already defined

Reply:
done